pcap/pcap.h

```diff
@@ -37,6 +37,10 @@
 
 #include <pcap/export-defs.h>
 
+#if defined(WIN32) && _MSC_VER < 1900
```

Comment:
We shouldn't be using snprintf() anywhere - we should be using pcap_snprintf().

_snprintf(), unfortunately, does *not* work the same way as C99 snprintf() - it does not guarantee that the resulting string is null-terminated, so we don't use it directly.  Instead, missing/win_snprintf.c has a pcap_vsnprintf() routine, which calls _vsnprintf() and then forcibly null-terminates the string; pcap_snprintf() uses pcap_vsnprintf().

So this change shouldn't be necessary.